src/components/manage/Contents/Contents.jsx

```diff
@@ -59,11 +59,20 @@ import {
   Pagination,
   Toolbar,
   Toast,
-  Icon as IconNext,
 } from '@plone/volto/components';
+import IconNext from '@plone/volto/components/theme/Icon/Icon';
```

Comment:
We are in the process of use Volto icons everywhere, so `<Icon >` component from Volto should be default, then I would start to name it as just `Icon`, then the "old" Semantic UI Icon component (in the meanwhile it's not completely removed from all the core views should be `IconOld`.

Reply:
Also, why not use the import from components?

Reply:
I didn't find any component name "Icon" in the component folder. Sorry! if I miss it.

Reply:
Sorry! I did not see the import of Icon in index.js. Imported icon from volto :)